src/CookieUtil.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Http\Message;
+
+final class CookieUtil
+{
+    /**
+     * Handles dates as defined by RFC 2616 section 3.3.1, and also some other
+     * non-standard, but common formats.
+     *
+     * @var array
+     */
+    private static $dateFormats = array(
+        'D, d M y H:i:s T',
```

Comment:
I add the format "Tuesday, 31 Mar 99 07:42:12 GMT" at the line 14 mentionned in https://github.com/php-http/client-common/pull/46